sql-basic-selects/src/scripts.sql

```diff
@@ -0,0 +1,34 @@
+1. SELECT * FROM city;
+2. SELECT name FROM city;
+3. SELECT name FROM city WHERE CountryCode = 'HUN';
+4. SELECT CountryCode FROM city ORDER BY CountryCode ASC;
```

Comment:
good - but please note that `ASC` is the default way of ordering, so writing it is kind of redundant

Reply:
also, you need to make sure there are no repetitions here: "Show all the country codes in alphabet order - **don’t repeat any of them**"